src/package.lisp

```diff
@@ -25,30 +25,30 @@
   (:use #:cl)
   (:export
    ;; Trie and its accessors
-   trie
-   key
-   value
-   activep
-   children
+   #:trie
+   #:key
+   #:value
+   #:activep
+   #:children
 
    ;; Trie generic functions
-   lookup
-   insert
-   find-node
-   remove-node
-   remove-index
-   mapkeys
-   mapvalues
-   all-keys
-   all-values
-   emptyp
-   clear
-   size
-   leafp
+   #:lookup
+   #:insert
+   #:find-node
+   #:remove-node
+   #:remove-index
+   #:mapkeys
+   #:mapvalues
+   #:all-keys
+   #:all-values
+   #:emptyp
+   #:clear
+   #:size
+   #:leafp
 
    ;; Trie utility functions
-   hash-table->trie
+   #:hash-table->trie
 
    ;; Conditions
-   empty-key-warning
-   wrong-key-type-error))
+   #:empty-key-warning
```

Comment:
I'm curious - what are the practical implications of this? What are we gaining by not interning the symbols?

Reply:
The way I see it is that avoiding interning symbols is just polite: interned symbols are never garbage-collected, they result in noise when you call `apropos`, and they can result in needless package conflicts.